covid-nlp/main.ipynb

```diff
@@ -0,0 +1,2384 @@
+{
```

Comment:
<p>same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>done</p>

Reply:
<p>Please put these example (and have them for all ie functions) in a cell bellow the function, using jupyter notebook to show the output,</p><p>there is no need to do this comment style markup when we are inside a notebook.</p>